iset.mm

```diff
@@ -85424,7 +85549,9 @@ nonnegative integers (cont.)". $)
   ${
     $d A a b x y z $.
     $( The image under negation of a bounded-above set of reals is bounded
-       below.  (Contributed by Paul Chapman, 21-Mar-2011.) $)
+       below.  For a theorem which is similar but also adds that the bounds
+       need to be the tightest possible, see ~ supinfneg .  (Contributed by
```

Comment:
Not a fan of that wording because it is supinfneg is about a least upper bound not just an upper bound (and likewise for greatest lower bound and lower bound).

Reply:
You're right. I meant: For a similar theorem about existence of infimum and sumremum, see ~ supinfneg .